apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -564,6 +564,37 @@ type Storage struct {
 // +kubebuilder:validation:Maximum=4094
 type VLANID int32
 
+type LLDPProcessed struct {
+	// https://github.com/openstack/python-ironic-inspector-client/blob/master/ironic_inspector_client/resource.py#L24-L59
```

Comment:
This becomes a comment from the Interface field, which is wrong. Ideally, we need a meaningful comment for each item, but definitely not a link to the code.